contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -12,22 +12,28 @@ import {UnstructuredStorage} from "@aragon/os/contracts/common/UnstructuredStora
 import {Math64} from "../lib/Math64.sol";
 import {MemUtils} from "../../common/lib/MemUtils.sol";
 import {MinFirstAllocationStrategy} from "../../common/lib/MinFirstAllocationStrategy.sol";
-import {SigningKeysStats} from "../lib/SigningKeysStats.sol";
+import {ILidoLocator} from "../../common/interfaces/ILidoLocator.sol";
+import {SigningKeys} from "../lib/SigningKeys.sol";
+import {Packed64x4} from "../lib/Packed64x4.sol";
 import {Versioned} from "../utils/Versioned.sol";
 
 interface IStETH {
     function sharesOf(address _account) external view returns (uint256);
     function transferShares(address _recipient, uint256 _sharesAmount) external returns (uint256);
+    function burnShares(address _account, uint256 _sharesAmount) external returns (uint256 newTotalShares);
```

Comment:
it has been removed already

please use `IBurner.requestBurnShares(adddress _account, uint256 _sharesAmount)` as a drop-in replacement

Reply:
yep